src/native/eventpipe/ds-ipc-posix.c

```diff
@@ -1,6 +1,7 @@
-#ifdef FEATURE_PERFTRACING_C_LIB_STANDALONE_PAL
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
 #define EP_NO_RT_DEPENDENCY
-#endif
 
 #include "ds-rt-config.h"
```

Comment:
Some legs are failing because of a missing header:
```suggestion
#include "ds-rt-config.h"
#include <string.h>
```

Reply:
I think it's because I removed the `FEATURE_PERFTRACING_C_LIB_STANDALONE_PAL` feature flag and the associated `ifndef`s.  I'm guessing when Mono builds, it didn't specify that flag.  I'll push a change.

Reply:
Maybe `FEATURE_PERFTRACING_C_LIB_STANDALONE_PAL` can stay deleted and conditions can just be based on `FEATURE_CORECLR`?

Reply:
I removed the `C_LIB` part of the feature flag, but left the feature flag in general since it does represent a feature that can be disabled.  Future runtime implementations might make use of this flag.